batteries/difftext/printdiff.luau

```diff
@@ -81,24 +180,56 @@ local function prettydiff(
 	b: string,
 	options: {
 		detailed: boolean?,
+		includeLineNumbers: boolean?,
 	}?
 ): string
-	if options then
-		if options.detailed then
-			return printDiffByLineDetailed(a, b)
-		end
+	if options and options.detailed then
+		return printDiffByLineDetailed(a, b, options.includeLineNumbers)
 	end
 
 	local result = {} :: { string }
 	local diff = myersdiffbyline(a, b)
-	for i, op in diff do
-		table.insert(
-			result,
-			if op.key == "ADD"
-				then richterm.brightGreen(`+ {op.text}`)
-				elseif op.key == "DELETE" then richterm.brightRed(`- {op.text}`)
-				else `{op.text}`
-		)
+	local maxNumLines = math.max(#a:split("\n"), #b:split("\n"))
```

Comment:
not loving that we do this split again after doing it in `myersdiffbyline`, maybe we should store the arrays in variables and call `myersdiff` directly?